src/components/calcite-video/calcite-video.scss

```diff
@@ -0,0 +1,176 @@
+:host-context([theme="dark"]) {
+  @include calcite-theme-dark();
+}
+:host {
+  --calcite-video-controls-on-hover-bg: rgba(255, 255, 255, 0.92);
+}
+:host([theme="dark"]) {
+  --calcite-video-controls-on-hover-bg: rgba(43, 43, 43, 0.92);
+}
+
+:host {
+  display: inline-flex;
+  flex-direction: column;
+  position: relative;
+  width: auto;
+  max-width: 100%;
+  background-color: var(--calcite-ui-foreground-1);
+  & video {
+    max-height: 100%;
+    max-width: 100%;
+  }
+
+  & calcite-loader {
+    z-index: 999;
+    position: absolute;
+    left: 0;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    margin: auto;
+  }
+  // todo hide these
+  video::-webkit-media-controls {
+    opacity: 0;
+  }
+  *::-webkit-media-controls-start-playback-button {
+    display: none !important;
+    -webkit-appearance: none;
+  }
+  *::-webkit-media-controls-panel {
+    display: none !important;
+    -webkit-appearance: none;
+  }
+}
+
+//focus
+:host {
+  @include focus-style-base();
+}
+:host(:focus) {
+  @include focus-style-outset();
+}
+// show controls on hover
+:host([show-controls-on-hover]) {
+  & .calcite-video-footer {
+    visibility: hidden;
+    position: absolute;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    opacity: 0;
+    transition: all 0.15s ease-in-out;
+    background-color: var(--calcite-ui-foreground-1);
+    box-sizing: border-box;
+    z-index: 9;
+  }
+}
+:host([show-controls-on-hover]:focus),
+:host([show-controls-on-hover]:focus-within),
+:host([show-controls-on-hover]:hover) {
+  .calcite-video-footer {
+    visibility: visible;
+    opacity: 1;
+  }
+}
+
+.calcite-video-wrapper {
```

Comment:
IMO I think we can move away from prefixing our shadow dom classes (ie. this could just be `.wrapper`). See the conventions doc: https://github.com/Esri/calcite-components/tree/master/conventions#css-class-names

Reply:
While adding subtitles... having slotted content inside a `<video>` that was in the TSX file caused a ton of issues, so I set to `shadow:false` for now and likely will need the namespace if that remains the case. I supposed we could take the slotted content and interpret and append inside the video out of the slot, but that is kind of an ugly workaround...

Reply:
Interesting, issues like the html elements stopped working when one was shadow and one was in the parent document?

Reply:
Since `video` was in the "light" 🌞  and the slotted content (`source` / `track`) were in the "shadow" 🌚 I was unable to actually render the video or act on the videos... 

<img width="410" alt="Screen Shot 2020-07-08 at 1 46 46 PM" src="https://user-images.githubusercontent.com/4733155/86968731-82c12900-c121-11ea-98d7-c256c5c2b250.png">

vs

<img width="411" alt="Screen Shot 2020-07-08 at 1 45 02 PM" src="https://user-images.githubusercontent.com/4733155/86968736-83f25600-c121-11ea-920a-ab7a504cc2d1.png">

Reply:
could have the default slot accept an entire video element, I guess. Like if you don't pass one we create it, but if you pass one we use that. Light DOM is fine too, though, you just open the video element up to CSS from the consuming page.